server/xpub/entities/manuscript/resolvers.test.js

```diff
@@ -189,6 +189,9 @@ describe('Submission', () => {
         Manuscript.statuses.MECA_EXPORT_PENDING,
       )
 
+      // Put in a delay to reduce risk of race condition
+      await new Promise(resolve => setTimeout(resolve, 100))
```

Comment:
I think better would be to assert that the value is either `SUCCEEDED` or `PENDING` since we don't actually care and that covers all possibilities.